extensions/amp-a4a/rtc-documentation.md

```diff
@@ -146,6 +151,51 @@ The RTC Response to a GET request must meet the following requirements:
     *   **<code>{"targeting": {"sport":["rugby","cricket"]}}</code>**</strong>
     *   The response body must be JSON, but the actual structure of that data need not match the structure here. Refer to Fast Fetch Network specific documentation for the required spec. (for example, if using DoubleClick, refer to DoubleClick docs).
 
+### RTC Error Pingback
+
+RTC supports sending a 1% sampling of RTC errors to specified errorReportingUrl's. For any given RTC callout URL, a corresponding errorReportingUrl may be specified, which will receive pings for, and only for, errors that resulted from the associated RTC callout. I.e., you can not specify one errorReportingUrl that receives batched pings for all RTC callouts from a page.
```

Comment:
Mention that is needs to be a secure URL and per page sampling here as well?

Reply:
done